### ISSUE TITLE
Something is broken in the alpine mirror

### ISSUE THREAD

**Issue by zacksiri** [lxc/lxc-ci#817]
So I ran into an odd issue today. I was trying to build alpine images locally. This used to work fine, I would be able to build images and launch them and everything would work normally.

Today I tried building and launching the image and there was no IP address issued. So I suspected maybe it's something to do with the original source of alpine.

I tried a different mirror `https://mirrors.lax.silicloud.com/alpine/` instead of the usual `https://mirror.csclub.uwaterloo.ca/alpine/` 

And managed to get a working image. I then checked the `last-updated` file and noticed that the silicloud mirror is behind the waterloo mirror. 

In cases like this how would we report the problem to the alpine developers?

**Comment by zacksiri** [comment]
This is very odd, I changed to any other mirror even with the same timestamp it works, it seems only the `https://mirror.csclub.uwaterloo.ca/alpine/` is broken

**Comment by stgraber** [comment]
Yeah, that's something that should be reported to Alpine upstream so they can look into it.

**Comment by zacksiri** [comment]
I've reported it https://gitlab.alpinelinux.org/alpine/aports/-/issues/15823

**Comment by stgraber** [comment]
We can close this one. Assuming we hit the same issue, the image will fail its tests on our side too and just be held up until they fix whatever is wrong in their repository.
